### DISCUSSION TITLE
Problem with scheduleEmail apiV2

### DISCUSSION THREAD

**Discussion by yrybakUIS** [fattureincloud/api#268] [Q&A] [answered]
Hello.
I get an error when sending an email `"L'indirizzo e-mail specificato non è stato configurato per l'invio."`

And I can't understand what the problem is. I am sending the following data
```

const apiInstance = new fattureInCloudSdk.IssuedDocumentsApi();
    const opts = {
      scheduleEmailRequest: {
        data: {
          sender_email: "testgmail@gmai.com",
          recipient_email: 'no-reply@fattureincloud.it',
          subject: "Test",
          body: "Hello world",
          include: {
            document: true,
            delivery_note: false,
            accompanying_invoice: true,
            attachment: true
          },
          send_copy: true,
          attach_pdf: true,
        }
      }
    }
    return apiInstance.scheduleEmail(this.companyId, id, opts);
```
Do you need to configure an email server for that?

**Comment by valmoz** [comment, marked as answer]
Hi @yrybakUIS 
if you want to send the email with our standard email address as the sender, you need to insert "no-reply@fattureincloud.it" as the sender (not the subscriber).
If instead, you want to send it from your custom email address, you must [setup your SMTP configuration](https://help.fattureincloud.it/help/articolo/128-configura-indirizzo-mail-come-mittente).

Let us know if it solves your issue.
Mauro
